cluster_orchestrator/cluster-manager/blueprints/service_blueprints.py

```diff
@@ -0,0 +1,106 @@
+import logging
+
+import services.service_operations as service_operations
+from bson import json_util
+from clients.mongodb_client import (
+    mongo_find_job_by_system_id,
+    mongo_update_job_status,
+)
+from clients.mqtt_client import mqtt_publish_edge_deploy
+from ext_requests.network_plugin_requests import network_notify_deployment
+from flask import Response, request
+from flask.views import MethodView
+from flask_smorest import Blueprint, abort
+from oakestra_utils.types.statuses import NegativeSchedulingStatus, PositiveSchedulingStatus
+
+# ........ Functions for job management ...............#
+# ......................................................#
+
+serviceblp = Blueprint(
+    "Multiple services operations",
+    "services",
+    url_prefix="/api/service",
+    description="Operations on services",
+)
+
+schedulingblp = Blueprint(
+    "Scheduling results",
+    "service",
+    url_prefix="/api/result",
+    description="Scheduling results operations",
+)
+
+
+@serviceblp.route("/<system_job_id>/<instance_number>")
+class ServiceController(MethodView):
+    @serviceblp.response(
+        200,
+        {"status": "ok"},
+        content_type="application/json",
+    )
+    def post(self, system_job_id, instance_number):
+        logging.info("Incoming Request /api/deploy")
+        job = request.json  # contains job_id and job_description
+
+        try:
+            service_operations.deploy_service(job, system_job_id, instance_number)
+        except Exception:
+            abort(500, "Failed to deploy service")
+
+        return Response(json_util.dumps({"status": "ok"}), mimetype="application/json")
+
+    @serviceblp.response(
+        200,
+        {"status": "ok"},
+        content_type="application/json",
+    )
+    def delete(self, system_job_id, instance_number):
+        """
+        find service in db and ask corresponding worker to delete task,
+        instance_number -1 undeploy all known instances
+        """
+        logging.info("Incoming Request /api/delete/ - to delete task...")
```

Comment:
```suggestion
        logging.info("Incoming Request /api/service/ - to delete task...")
```
Since the path has been changed we should change the logging I think